src/types/types.ts

```diff
@@ -19,6 +21,14 @@ export const typeDefs = gql`
     email: String!
     birthDate: String!
   }
+  input LoginInput {
+    email: String!
+    password: String!
+  }
+  type UserToken {
```

Comment:
```suggestion
  type Login {
```

Acho que expressa melhor a dupla user + token.